node/directives/helper.ts

```diff
@@ -29,7 +29,12 @@ export const validateAdminToken = async (
       hasCurrentValidAdminToken = true
 
       if (authUser?.audience === 'admin') {
-        hasValidAdminToken = true
+        const hasAdminPermissions = await lm.getUserAdminPermissions(
+          authUser.account,
+          authUser.id
+        )
+
+        hasValidAdminToken = hasAdminPermissions
```

Comment:
can we remove this line by setting the result of line above directly to `hasValidAdminToken`?